estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,13 @@
+from odoo import fields, models
+
+
+class PropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Offers in real estate"
+
+    price = fields.Float(string="Price", required=True)
+    status = fields.Selection(
+        [("accepted", "Accepted"), ("refused", "Refused")], string="Status", copy=False
+    )
```

Comment:
```suggestion
    status = fields.Selection(
        string=[
            ("accepted", "Accepted"), 
            ("refused", "Refused")
        ], 
        string="Status", 
        copy=False
    )
```
Maintain code consistency